src/widget.js

```diff
@@ -350,6 +354,7 @@ window.RocketChat.livechat = {
 	// methods
 	pageVisited,
 	setCustomField,
+	setCustomTitle,
```

Comment:
My suggestion is to use the `setTheme` method to get a new parameter, such as the title.

Reply:
Oh sorry,
I though that theme property is only used for color or bgcolor only. 
Anyway, thanks :) 
I'll update it asap